src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -218,3 +218,11 @@ def do_collide_ssns(
         randomness=randomness,
     )
     return obs_data
+
+
+def copy_ssn_from_household_member(
```

Comment:
I'd move this into make_results.py since it's only called there. And then if we were to start calling it elsewhere I'd move into utilities.py